packages/i18n/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@nodejs/i18n",
```

Comment:
I'm not a fan of this scope, we should have one specific to the website because this one is also used apr undici. The packages in `nodejs` are user packages, so it's not the best idea. But there's also `@node-core`, which doesn't fit either because it's intended for node core 😁. 
So for me it would be necessary to have a scope specific to the website project.

Reply:
I think it should be "@node-core/website-i18n" and the current "@nodejs/website" on (apps/site) should be "@node-core/website"

Reply:
i am fine with that - one thing to keep in mind is that we cannot anticipate how people will use these packages once split out. `@node-core/website-i18n` could suddenly become very useful to someone downstream. That's not to say we should optimize for unknown, but I do like re-use.